src/matter/ControllerNode.ts

```diff
@@ -135,6 +117,30 @@ class Controller implements GeneralNode {
         });
     }
 
+    async applyConfiguration(config: MatterControllerConfig, isInitialization = false): Promise<MessageResponse> {
```

Comment:
we could add some JSDOC to newly created methods, what do you think?

Reply:
I put jsdoc generally on my list ,so separate PR, but agreed